### DISCUSSION TITLE
RN 0.74 and the new --experimental-debugger

### DISCUSSION THREAD

**Discussion by ashtar23** [nrwl/nx#27074] [Questions]
Hello, 

Hopefully this hasn't been already asked, tried going through some of the opened and closed tickets, and haven't found anything similar.

I've been trying to enable and test the new experimental debugger in a Nx workspace but haven't found any clear instructions on how to do so. According to the [official docs](https://reactnative.dev/docs/debugging), enabling the new debugger requires running:

`npx react-native start --experimental-debugger`

For instance, my usual command to run an iOS debug build with Nx is:

`yarn ios --simulator "iPhone 15 Pro"`
This translates to:
`nx run-ios [app-name]`

Here's the relevant config for `run-ios` in my `project.json`:

```
"run-ios": {
			"executor": "@nx/react-native:run-ios",
			"defaultConfiguration": "integration",
			"options": {
				"port": 8080
			},
			"configurations": {
				"integration": {
					"scheme": "[app-name]"
				},
				"staging": {
					"scheme": "[app-name]"
				},
				"production": {
					"scheme": "[app-name]"
				}
			},
			"dependsOn": ["sync-deps", "pod-install", "i18n", "storybook-generator"]
		},
```

**Comment by phazei** [comment]
Same, I'd say it's a bug that it doesn't allow this.

**Comment by phazei** [comment]
I was able to run it with the experimental debugger with this command added to project.json:
```
        "start": {
            "executor": "@nx/react-native:start",
            "options": {
                "port": 8081
            },
            "configurations": {
                "clean": {
                    "resetCache": true
                },
                "experimental-debugger": {
                    "experimental-debugger": true
                }
            },
            "dependsOn": [ "sync-deps" ]
        }
```

**Reply by ashtar23** [reply]
That's awesome! Gonna give it a go
